packages/manager/modules/config/package.json

```diff
@@ -32,5 +32,8 @@
   },
   "devDependencies": {
     "@ovh-ux/component-rollup-config": "^8.0.0 || ^9.0.0"
+  },
+  "peerDependencies": {
+    "@ovh-ux/manager-models": "^1.0.0"
```

Comment:
As it is not an angularJS dependency, it could be added in `dependencies` I think

Reply:
Fixed.